README.md

```diff
@@ -1,194 +1,239 @@
+
+<!-- Regenerate with "pre-commit run -a markdown-toc" -->
+
+<!-- tocstop -->
 
 ## Overview
 
-MilMove is a system to help service members (and other authorized personnel) move their gear and possessions from one
-place to another. This codebase has been written to perform load tests on the MilMove app for the purpose of gathering
+MilMove is a system to help service members (and other authorized personnel) move their gear and
+possessions from one place to another.
+
+This codebase has been written to perform load tests on the MilMove app for the purpose of gathering
 data about responses times, finding breakpoints, and assessing the overall health of the system.
 
+## Project Directories
+
+This section covers some high-level notes for some directories included in this repo.
+
+### `ecs/`
+
+This directory contains a representation of the task definition for running the docker container in
+AWS. To make changes to the task definition will require changing the terraform in
+`transcom/transcom-infrasec-gov-nonato/transcom-gov-dev/app-dev/loadtesting.tf`. This file is
+updated manually to reflect the current state.
+
 ### `locustfiles/`
 
-[Locust](https://docs.locust.io/en/stable/) uses a python file called a "locustfile" as the base for running a load
-test. This directory contains all of the locustfiles for this repo. This file must contain at least one class definition
-that inherits from the locust `User` class. Locust will dynamically create instances of these `User` classes to simulate
-the request load desired.
+[Locust](https://docs.locust.io/en/stable/) uses a python file called a "locustfile" as the base for
+running a load test. This directory contains all locustfiles for this repo. This file must contain
+at least one class definition that inherits from a locust `User` class (or more likely a subclass).
+Locust will dynamically create instances of these `User` classes to simulate the request load
+desired.
 
-Each of these files can be thought of as a different test case for the system, although locust also provides a number of
-[config options](https://docs.locust.io/en/stable/configuration.html) to allow you to manipulate which users and/or
-tasks run from any given locustfile.
+Each of these files can be thought of as a different test case for the system, although locust also
+provides a number of [config options](https://docs.locust.io/en/stable/configuration.html) to allow
+you to manipulate which users and/or tasks run from any given locustfile.
 
-### `tasks/`
+In particular, our `locustfiles/all.py` combines all our `User` classes into a single run.
 
-Each `User` class needs a set of tasks to complete to be able to run a load test. All tasks are callables that can be
-manually set into the `tasks` attribute of the user, or they can be organized into instances of the locust class
-`TaskSet` and then associated with a user. This directory contains all of the tasks used in our load tests.
+### `scripts`
```

Comment:
just for consistency this is the only folder without a trailing slash `scripts/`

Reply:
oh good catch, thanks!